fullstop-it/src/test/java/org/zalando/stups/fullstop/whitelist/WhitelistRulesEvaluatorTest.java

```diff
@@ -68,7 +67,7 @@ public void testImageOwner() throws Exception {
     public void testApplicationID() throws Exception {
         Map<String, String> metainfo = newHashMap();
         metainfo.put("application_id", "myApp");
-        violationEntity = new ViolationEntity(null, null, null, null, metainfo, null, null, null);
+        violationEntity = new ViolationEntity(null, null, null, null, metainfo, null, null, null, null, null);
```

Comment:
I would expect the white-listing rules to have changed and match app id by the new property now